`make ghcid` fails

## Description

`make ghcid` fails with
```
Warning: Some targets (purescript-cst) are not local packages, and so cannot be directly loaded. In
         future versions of stack, this might be supported - see
         https://github.com/commercialhaskell/stack/issues/1441 . It can still be useful to specify
         these, as they will be passed to ghci via -package flags.
The following GHC options are incompatible with GHCi and have not been passed to it: -threaded -O2 -Werror
Configuring GHCi with the following packages: purescript

* * * * * * * *

Warning: Multiple files use the same module name:
         * Paths_purescript found at the following paths
           * /Users/pete/Code/purescript/purescript/.stack-work/dist/aarch64-osx/Cabal-3.6.3.0/build/autogen/Paths_purescript.hs (purescript:lib)
           * /Users/pete/Code/purescript/purescript/.stack-work/dist/aarch64-osx/Cabal-3.6.3.0/build/purs/autogen/Paths_purescript.hs (purescript:exe:purs)
           * /Users/pete/Code/purescript/purescript/.stack-work/dist/aarch64-osx/Cabal-3.6.3.0/build/tests/autogen/Paths_purescript.hs (purescript:test:tests)
* * * * * * * *

GHCi, version 9.2.3: https://www.haskell.org/ghc/  :? for help
<command line>: cannot satisfy -package purescript-cst
    (use -v for more information)
Command "stack ghci purescript:exe:purs purescript:lib purescript:test:tests purescript-cst --main-is purescript:exe:purs --ghci-options -fno-code" exited unexpectedly with error message:     (use -v for more information)
make: *** [Makefile:22: ghcid] Error 1
```
I'm new to poking around this repo, but it seems that the `cannot satisfy -package purescript-cst` bit is probably because that package no longer exists after https://github.com/purescript/purescript/pull/4290.

## To Reproduce

Run `make ghcid` from the root directory of this repo

## Expected behavior

I would expect `ghcid` to run as specified here https://github.com/ndmitchell/ghcid#what-you-get

## Additional context

Add any other context about the problem here.

## PureScript version

`dev`

--- a/CHANGELOG.d/internal_remove-purescript-cst-from-makefile.md
+++ b/CHANGELOG.d/internal_remove-purescript-cst-from-makefile.md
@@ -1,0 +1,1 @@
+* Remove `purescript-cst` from Makefile

--- a/Makefile
+++ b/Makefile
@@ -19,10 +19,10 @@ help: ## Print documentation
 	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
 
 ghcid: ## Run ghcid to quickly reload code on save.
-	ghcid --command "stack ghci purescript:exe:purs purescript:lib purescript:test:tests purescript-cst --main-is purescript:exe:purs --ghci-options -fno-code"
+	ghcid --command "stack ghci purescript:exe:purs purescript:lib purescript:test:tests --main-is purescript:exe:purs --ghci-options -fno-code"
 
 ghcid-test: ## Run ghcid to quickly reload code and run tests on save.
-	ghcid --command "stack ghci purescript:lib purescript:test:tests purescript-cst --ghci-options -fobject-code" \
+	ghcid --command "stack ghci purescript:lib purescript:test:tests --ghci-options -fobject-code" \
 	    --test "Main.main"
 
 build: ## Build the package.

